src/states/elder/mod.rs

```diff
@@ -2090,22 +2106,35 @@ impl Approved for Elder {
 
         if self_sec_update {
             self.chain.reset_candidate_if_member_of(sec_info.members());
+
+            // Vote to update our self messages proof
+            self.vote_send_section_info_ack(SendAckMessagePayload {
```

Comment:
Is this a new requirement? Previously we only send out section_info ack for non-self-section.

Reply:
Yes it is a new requirement so message to self do not need proof since genesis.
If you look at the commit history on this PR, one of the commit is making it so we send to all. Also PR message:
```
Ensure we update their_keys/their_knowledge for ourselves so we can trust message from self section, and send minimal proof to self section: prune proof similarly when we receive an AckMessage.
```